src/python_bindings/bindings.cpp

```diff
@@ -156,7 +156,57 @@ PYBIND11_MODULE(desbordante, module) {
             .def("get_num_error_rows", &PyFDVerifier::GetNumErrorRows)
             .def("get_highlights", &PyFDVerifier::GetHighlights);
 
-    DEFINE_ALGORITHM(DataStats, Algorithm).def("get_result_string", &PyDataStats::GetResultString);
+    DEFINE_ALGORITHM(DataStats, Algorithm)
+            .def("get_result_string", &PyDataStats::GetResultString)
+            .def("get_number_of_values", &PyDataStats::GetNumberOfValues,
+                 "Get number of values in column.")
+            .def("get_number_of_columns", &PyDataStats::GetNumberOfColumns,
+                 "Get number of columns in table.")
+            .def("get_null_columns", &PyDataStats::GetNullColumns,
+                 "Get indices of columns with only null values.")
+            .def("get_columns_with_null", &PyDataStats::GetColumnsWithNull,
+                 "Get indices of columns which contain null value.")
+            .def("get_columns_with_all_unique_values", &PyDataStats::GetColumnsWithUniqueValues,
+                 "Get indices of columns where all values are distinct.")
+            .def("get_number_of_distinct", &PyDataStats::Distinct,
+                 "Get number of unique values in the column.")
+            .def("is_categorical", &PyDataStats::IsCategorical,
+                 "Check if quantity => number of unique values in the column.")
+            .def("show_sample", &PyDataStats::ShowSample, py::arg("start_row"), py::arg("end_row"),
+                 py::arg("start_col"), py::arg("end_col"), py::arg("str_len") = 10,
+                 py::arg("unsigned_len") = 5, py::arg("double_len") = 10,
+                 "Returns table slice from start_row to end_row and from start_col to end_col. "
+                 "Data values converted to string type.")
```

Comment:
```suggestion
                 "Returns a table slice containing values from rows in the range [start_row, end_row] and columns in the range [start_col, end_col]. Data values are converted to strings.")
```
Split into lines however is correct